### PR TITLE
read in other available sensors

### PR DESCRIPTION
**Pull request by m-patterson-wearable**
read in other available sensors from api.

Copied the sensor names from api documentaiton

### CODE REVIEW

File: actiapi/v3.py [hunk 1]
```diff
@@ -58,7 +58,9 @@ def get_files(
         end: Optional[str] = None,
         data_format: Literal["avro", "csv"] = "avro",
         sensor: Literal[
-            "raw-accelerometer", "temperature", "ppg-green"
+            "raw-accelerometer", "temperature",
+            "ppg-green (25hz)", "ppg-green-100-hz",
```
Comment (rouzbeh) [inline actiapi/v3.py:62]: I think the name is "ppg-green", not "ppg-green (25hz)".

**Review by rouzbeh — CHANGES_REQUESTED** [review]
I think the name is not correct. I have tested "ppg-green" and know it works.

**Comment by m-patterson-wearable** [comment]
Documentation in the AG API page says this:
Screen Shot 2024-03-26 at 4 27 45 PM

https://github.com/actigraph/CentrePoint3APIDocumentation/blob/main/sections/data_access_files.md

**Comment by rouzbeh** [comment]
> Documentation in the AG API page says this: Screen Shot 2024-03-26 at 4 27 45 PM
> 
> https://github.com/actigraph/CentrePoint3APIDocumentation/blob/main/sections/data_access_files.md

Yes, but that is poorly worded. `ppg-green` gives you 25Hz PPG data, as opposed to `ppg-green-100-hz`. They first created `ppg-green` before they realized there would be other frequencies.

**Review by rouzbeh — APPROVED** [review]

**Pull request merged**